backend/app/assets/javascripts/spree/backend/promotions.js

```diff
@@ -147,7 +147,7 @@ function initProductActions () {
       var quantity = $container.find('input[name="add_quantity"]').val()
       if (variant_id) {
         // Add to the table
-        var newRow = '<tr><td>' + productName + '</td><td>' + quantity + '</td><td><img src="/assets/admin/icons/cross.png"/></td></tr>'
+        var newRow = '<tr><td>' + productName + '</td><td>' + quantity + '</td><td><img src="/assets/admin/icons/cross.png"/>KKK</td></tr>'
```

Comment:
Is there a specific reason of putting `KKK` here? Also 
```suggestion
        var newRow = '<tr><td>' + product_name + '</td><td>' + quantity + '</td><td><img src="/assets/admin/icons/cross.png"/>KKK</td></tr>'
```

Is there a specific reason of putting `KKK` here?

Reply:
@kaliszKrzysztof I was looking for that missing png in the DOM, I couldn't find it so just swapped it out for a span bases icon.